go_optouts/store/memory.py

```diff
@@ -1,9 +1,31 @@
-from zope.interface import implements
-from interface import IOptOutStore
 import uuid
 
+from zope.interface import implements
+
+from .interface import IOptOutBackend, IOptOutCollection
```

Comment:
Is there a reason for the relative import here, rather than `from go_optouts.store.interface import ...`